From 039baa44a57cd9069134019a6f6628cc6b6a70e5 Mon Sep 17 00:00:00 2001
From: Kain Highwind <highwind.1977@gmail.com>
Date: Fri, 24 Jul 2015 23:40:32 +0200
Subject: [PATCH] disable build

Change-Id: I6d860d9bba2a3077f26f3840d6649844f65e3272
---
 core/Makefile | 1 -
 1 file changed, 1 deletion(-)

diff --git a/core/Makefile b/core/Makefile
index b5cdc66..dd5acf3 100644
--- a/core/Makefile
+++ b/core/Makefile
@@ -1709,7 +1709,6 @@ $(INTERNAL_OTA_PACKAGE_TARGET): $(BUILT_TARGET_FILES_PACKAGE) $(DISTTOOLS)
 	@echo -e ${CL_YLW}"Package OTA:"${CL_RST}" $@"
 	$(hide) MKBOOTIMG=$(MKBOOTIMG) \
 	   $(OTA_FROM_TARGET_SCRIPT) -v \
-	   --block \
 	   $(if $(WITH_LZMA_OTA), -z) \
 	   -p $(HOST_OUT) \
 	   -k $(KEY_CERT_PAIR) \
-- 
1.9.1

